broker/cloud_run/lsst/classify_snn/deploy.sh

```diff
@@ -10,109 +10,84 @@ teardown="${2:-False}"
 # name of the survey this broker instance will ingest
 survey="${3:-lsst}"
 region="${4:-us-central1}"
-# get environment variables
+# get the environment variable
 PROJECT_ID=$GOOGLE_CLOUD_PROJECT
-PROJECT_NUMBER=$(gcloud projects describe "$PROJECT_ID" --format="value(projectNumber)")
 
 MODULE_NAME="supernnova"  # lower case required by cloud run
 ROUTE_RUN="/"  # url route that will trigger main.run()
 
-# function used to define GCP resources; appends testid if needed
 define_GCP_resources() {
     local base_name="$1"
+    local separator="$2"
```

Comment:
Since we prefer "-" and only use "_" when required by GCP naming rules, you could set "-" as the default here and then most of the calls below wouldn't have to send this arg. That would make things less error prone, especially for future developers less familiar with our naming preferences..